<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BTVRCMInterface</class>
 <widget class="QWidget" name="BTVRCMInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>962</width>
    <height>575</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>962</width>
    <height>575</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>962</width>
    <height>603</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	font-size: 16px;
	font-family: Microsoft YaHei;
	color: black;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_32">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>血温血容量板：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_btvBoard_hardVer">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>H-V6.00</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_btvBoard_softVer">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>S-V6.00</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_28">
       <property name="text">
        <string>通讯开关：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CustomSwitchButton" name="comSwitchBtn_btvBoard" native="true">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_33">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>实时清除率板：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_rcmBoard_hardVer">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>H-V6.00</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_rcmBoard_softVer">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>S-V6.00</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>通讯开关：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CustomSwitchButton" name="comSwitchBtn_rcmBoard" native="true">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget_center" native="true">
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border: 1px solid rgb(51, 204, 255);
	border-radius: 5px;
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(234, 255, 255, 200), stop:1 rgba(253, 231, 231, 150));
	color: rgb(0, 51, 102);
}
QPushButton:hover{
	border: 1px solid rgb(51, 204, 255);
	border-radius: 5px;
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(234, 255, 255, 255), stop:1 rgba(253, 231, 231, 255));
}
QPushButton:pressed{
	border: 1px solid rgb(51, 204, 255);
	border-radius: 5px;
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(214, 235, 235, 200), stop:1 rgba(243, 211, 211, 150));
}
QPushButton:checked{
	border: 1px solid rgb(0, 224, 0);
	border-radius: 5px;
	background: rgba(153, 255, 204, 100);
}

QGroupBox{
	border: 1px solid rgb(28, 134, 238);
	border-radius: 4px;
	margin-top: 0.5em;
}
QGroupBox::title{
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 1em;
}
</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>血温血容量传感器</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>静脉血温
传感器：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="VTStatusBtn">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>38</height>
              </size>
             </property>
             <property name="text">
              <string>正常</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>动脉血温
传感器：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="ATStatusBtn">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>38</height>
              </size>
             </property>
             <property name="text">
              <string>正常</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="2">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>静脉血温：
(°C)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="VTBtn">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>38</height>
              </size>
             </property>
             <property name="text">
              <string>0.00</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="3">
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="label_12">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>动脉血温：
(°C)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="ATBtn">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>38</height>
              </size>
             </property>
             <property name="text">
              <string>0.00</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>血容量
传感器：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="BVStatusBtn">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>38</height>
              </size>
             </property>
             <property name="text">
              <string>正常</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>血液：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bloodDetectBtn">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>38</height>
              </size>
             </property>
             <property name="text">
              <string>未检测到</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="2">
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>监测盖:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="BVCoverBtn">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>38</height>
              </size>
             </property>
             <property name="text">
              <string>关闭</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="3">
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_10">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>近红外LED:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="LEDSWBtn">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>38</height>
              </size>
             </property>
             <property name="text">
              <string>关闭</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>受光电压：
(mV)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="BV_VBtn">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>38</height>
              </size>
             </property>
             <property name="text">
              <string>否</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QLabel" name="label_14">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>血容量：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="BVBtn">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>38</height>
              </size>
             </property>
             <property name="text">
              <string>0</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="2">
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>初始血容量：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="BV0Btn">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>38</height>
              </size>
             </property>
             <property name="text">
              <string>0</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="3">
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>相对血容量：
(%)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="RBVBtn">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>38</height>
              </size>
             </property>
             <property name="text">
              <string>0.00</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>实时清除率监测传感器</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_18">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_14">
           <item>
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>传感器校正：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="RCMCorrectBtn">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>38</height>
              </size>
             </property>
             <property name="text">
              <string>未校正</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_15">
           <item>
            <widget class="QLabel" name="label_17">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>传感器状态：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="RCMStatusBtn">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>38</height>
              </size>
             </property>
             <property name="text">
              <string>正常</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_16">
           <item>
            <widget class="QLabel" name="label_18">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>检测状态：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="RCMDetectBtn">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>38</height>
              </size>
             </property>
             <property name="text">
              <string>停止</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_17">
           <item>
            <widget class="QLabel" name="label_19">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>吸光度：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="absBtn">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>38</height>
              </size>
             </property>
             <property name="text">
              <string>0.00</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>321</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomSwitchButton</class>
   <extends>QWidget</extends>
   <header>interface/commonItem/customswitchbutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
